c7n/resources/efs.py

```diff
@@ -57,10 +57,12 @@ class resource_type(TypeInfo):
         enum_spec = ('describe_mount_targets', 'MountTargets', None)
         permission_prefix = 'elasticfilesystem'
         name = id = 'MountTargetId'
-        filter_name = 'MountTargetId'
-        filter_type = 'scalar'
         arn = False
         cfn_type = 'AWS::EFS::MountTarget'
+        supports_trailevents = True
+    
+    def get_resources(self, ids, cache=True):
+        return super(ElasticFileSystemMountTarget, self).get_resources(ids, cache)
```

Comment:
We should be able to remove this if it's only handing off to the parent class's `get_resources()` anyway.